e2e/command_test.go

```diff
@@ -599,6 +615,83 @@ func TestE2E_CreateTaskCommand_NoTaskFileProvided(t *testing.T) {
 	}
 }
 
+// TestE2E_CreateDeleteCreateTrigger tests that after creating a task, then deleting
+// the task, and then finally re-creating the task, the task will trigger
+func TestE2E_CreateDeleteCreateTrigger(t *testing.T) {
+	t.Parallel()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "create_cmd")
```

Comment:
I think we try to keep these tempDir values unique. Looks like "create_cmd" is already used by `TestE2E_CreateTaskCommand`

Reply:
copypasta, will fix